examples/03-tips-n-tricks/04-rotational-displacement.py

```diff
@@ -0,0 +1,126 @@
+"""
+.. _ref_rotational_displacement_example:
+
+Generating and Extracting Rotational Displacement
+-------------------------------------------------
+
+Not all element types have rotational degrees of freedom, but generally,
+"shell" ones do. In this example we create a square shell with thickness
+of 0.1 and bend it, creating rotational displacement.
+
+We subsequently plot the cumulative principal stresses and use
+:class:`ansys.mapdl.core.inline_functions.Query` to
```

Comment:
Good work with the class reference.

Reply:
Turns out class reference isn't linking because we don't specifically call out Query within out API documentation.  Docs need to be refactored using the custom class template anyway, so I'll add this in another PR.